store/storetest/channel_store.go

```diff
@@ -563,36 +563,28 @@ func testChannelStoreGetByName(t *testing.T, ss store.Store) {
 	o1.Type = model.CHANNEL_OPEN
 	store.Must(ss.Channel().Save(&o1, -1))
 
-	r1 := <-ss.Channel().GetByName(o1.TeamId, o1.Name, true)
-	if r1.Err != nil {
-		t.Fatal(r1.Err)
-	} else {
-		if r1.Data.(*model.Channel).ToJson() != o1.ToJson() {
-			t.Fatal("invalid returned channel")
-		}
-	}
-
-	if err := (<-ss.Channel().GetByName(o1.TeamId, "", true)).Err; err == nil {
-		t.Fatal("Missing id should have failed")
-	}
+	result := <-ss.Channel().GetByName(o1.TeamId, o1.Name, true)
+	require.NotNil(t, result)
+	assert.Nil(t, result.Err)
```

Comment:
This will be a require, if and maybe the previous `require.NotNil` is not needed because you don't receive a error, the result have the expected value.